simulator/docs/integrate-your-scheduler.md

```diff
@@ -1,13 +1,10 @@
 ## Integrate your scheduler 
 
-There are several ways to integrate your scheduler into the simulator.
+You can integrate your scheduler into the simulator.
 
-Basically you have two options on the table:
-- Integrate your scheduler plugins into the scheduler running in the simulator
-  - Check out [custom-plugin.md](./custom-plugin.md) 
-- Disable the scheduler running in the simulator and use your scheduler instead
-  - We call this feature "external scheduler"
-  - Check out [external-scheduler.md](./external-scheduler.md)
+To add your custom scheduler plugins, refer to the following resources:
+- [debuggable-scheduler.md](./debuggable-scheduler.md#integrate-your-plugins-to-the-simulator) for basic integration steps.
+- [custom-plugin.md](./custom-plugin.md) for more detailed guidance on configuring out-of-tree plugins.
```

Comment:
Ah, actually, we can remove `custom-plugin.md` as well because users can just register their custom plugins via the debuggable scheduler.

So, here can be simplified like:

```suggestion
To add your custom scheduler plugins, see [debuggable-scheduler.md](./debuggable-scheduler.md#integrate-your-plugins-to-the-simulator).
```

And, also please remove the page `custom-plugin.md`.